bin/migrate.js

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env node
+
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+// use this script to call our pg-patcher wrapper via the command line.
+// TODO add 'next' and 'prev' convenience methods?
+// TODO bring back commander, I guess
+
+var fs = require('fs');
+var log = require('../server/logger')('bin.migrate');
+var migrator = require('../server/db/migrator.js');
+var targetLevel = parseInt(process.argv[2], 10);
+
+// if no args were passed, or if the arg wasn't an integer,
+// then, by default, use the highest defined patch level
+if (isNaN(targetLevel)) {
+  files = fs.readdirSync(__dirname + '/../server/db/migrations');
```

Comment:
actually, I'll file a followup bug to audit our codebase for windows compatibility issues